### PR TITLE
Topics remapped, namespace added

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: sr_gui_servoing/src/sr_gui_servoing/displaying.py [hunk 1]
```diff
@@ -18,11 +18,11 @@ class DisplayImage(object):
     def __init__(self, node_name):
 
         rospy.init_node(node_name)
-        self.color = rospy.get_param('/color')
+        self.color = rospy.get_param('/tracking/color')
```
Comment (ugocupcic) [inline sr_gui_servoing/src/sr_gui_servoing/displaying.py]: no `/` at the beginning

File: sr_gui_servoing/src/sr_gui_servoing/displaying.py [hunk 2]
```diff
@@ -18,11 +18,11 @@ class DisplayImage(object):
...
         self.cv_window_name = 'Video'
 
         # Initialize a number of global variables
-        self.smin = rospy.get_param('/saturation_min')
+        self.smin = rospy.get_param('/tracking/saturation_min')
```
Comment (ugocupcic) [inline sr_gui_servoing/src/sr_gui_servoing/displaying.py]: same

File: sr_gui_servoing/src/sr_gui_servoing/displaying.py [hunk 3]
```diff
@@ -37,14 +37,18 @@ def __init__(self, node_name):
 
         self.utils = Utils()
 
-        self.image_sub = rospy.Subscriber('/camera/rgb/image_color', Image, self.display)
-        self.roi_sub = rospy.Subscriber("/roi/track_box", RegionOfInterest, self.roi_callback)
+        self.image_sub = rospy.Subscriber('/camera/rgb/image_color', Image,
```
Comment (ugocupcic) [inline sr_gui_servoing/src/sr_gui_servoing/displaying.py]: same

File: sr_object_segmentation/scripts/segmentation.py [hunk 4]
```diff
@@ -12,20 +12,24 @@
 class Segmentation(object):
     def __init__(self):
 
-        self.color = rospy.get_param('/color')
+        self.color = rospy.get_param('/tracking/color')
```
Comment (ugocupcic) [inline sr_object_segmentation/scripts/segmentation.py]: same

File: sr_object_segmentation/scripts/segmentation.py [hunk 5]
```diff
@@ -12,20 +12,24 @@
...
 
         self.seg = HSVSegmentation(self.color)
         self.utils = Utils()
 
-        self.image_sub = rospy.Subscriber('/camera/rgb/image_color', Image, self.image_callback)
+        self.image_sub = rospy.Subscriber('/camera/rgb/image_color', Image,
```
Comment (ugocupcic) [inline sr_object_segmentation/scripts/segmentation.py]: again

File: sr_object_tracking/launch/tracking.launch [hunk 6]
```diff
@@ -4,17 +4,21 @@
       <remap from="/image_raw" to="/camera/rgb/image_color"/>
   </node>
 
-  <node name="segmentation" pkg="sr_object_segmentation" type="segmentation.py"  />
+  <group ns="tracking">
```
Comment (ugocupcic) [inline sr_object_tracking/launch/tracking.launch]: why are you using the namespace?

File: sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py [hunk 7]
```diff
@@ -16,8 +16,9 @@ def __init__(self):
         self.utils = Utils()
 
         # Initialize a number of global variables
-        self.smin = rospy.get_param('/saturation_min')
-        self.size = rospy.get_param('/size')
+        self.smin = rospy.get_param('/tracking/saturation_min')
```
Comment (ugocupcic) [inline sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py]: don't begin with `/`, this is the tracking node right? so it should just be `~/saturation_min`

File: sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py [hunk 8]
```diff
@@ -16,8 +16,9 @@ def __init__(self):
...
+        self.size = rospy.get_param('/tracking/size')
```
Comment (ugocupcic) [inline sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py]: `~/size`

File: sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py [hunk 9]
```diff
@@ -16,8 +16,9 @@ def __init__(self):
...
+        self.color = rospy.get_param('/tracking/color')
```
Comment (ugocupcic) [inline sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py]: `~/color`

File: sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py [hunk 10]
```diff
@@ -41,16 +41,23 @@ def __init__(self):
         self.upper = np.array(self.upper, dtype="uint8")
 
         # Subscribe to the image topic and set the appropriate callback
-        self.image_sub = rospy.Subscriber('/camera/rgb/image_color', Image, self.image_callback)
-        self.selection_sub = rospy.Subscriber("/roi/segmented_box", RegionOfInterest, self.selection_callback)
-        self.selection_sub = rospy.Subscriber("/roi/selection", RegionOfInterest, self.selection_callback)
+        self.image_sub = rospy.Subscriber('/camera/rgb/image_color', Image,
```
Comment (ugocupcic) [inline sr_object_tracking/src/sr_object_tracking/sr_object_tracking.py]: no `/`

File: sr_object_tracking/launch/tracking.launch (hunk 6 above)
Comment (ghost) [inline sr_object_tracking/launch/tracking.launch]: I thought it was a good idea to isolate my program..

**Pull request merged**